hack/tools.go

```diff
@@ -0,0 +1,8 @@
+// +build tools
+
+package tools
+
+import (
+	_ "knative.dev/test-infra"
+	_ "knative.dev/test-infra/scripts"
```

Comment:
I think this is all we need from test-infra so the line above is redundant, right?

Shouldn't we also import the tests from `serving`? Something like `_ "knative.dev/serving/test/conformance/ingress"` ?

Reply:
Yeah, `test-infra` isn't needed, I'm actually going to try to remove that `dummy.go` file upstream.

I'll play whack-a-mole with the e2e tests next.

Reply:
Ok, I copied over the list of `required` pieces from net-contour, but to unblock the test images we need this to land: https://github.com/knative/test-infra/pull/1887. cc @chizhg

Reply:
Build tests failed (as expected until the test-infra change), so if integration tests pass, then that should be all that's blocking this 🤞

Reply:
there we go 💪

Reply:
knative/test-infra#1887 has been merged. Can you update the PR and see if the tests pass? :pray: